core-bundle/src/Resources/contao/library/Contao/File.php

```diff
@@ -879,12 +879,6 @@ protected function getIcon()
 	 */
 	protected function getHash()
 	{
-		// Do not try to hash if bigger than 2 GB
-		if ($this->filesize >= 2147483648)
-		{
-			return '';
-		}
-
 		return md5_file($this->strRootDir . '/' . $this->strFile);
```

Comment:
Wouldn't this still cause performance problems though?

Reply:
@ausi performed some tests yesterday and could not find any downsides. There seemed to be an old bug on windows with files > 2GB, but I guess that is resolved in the meantime.

Reply:
The performance I measured responds linear to the file size. So there is no difference between ten 1GB files and one 10GB file AFAIK.

Reply:
> There seemed to be an old bug on windows with files > 2GB, but I guess that is resolved in the meantime.

You guess or should I check? 😁

Reply:
I could find no issues (other than the hash calculation taking 4500 ms for a 2.5 GiB test file in my case).

Reply:
Thanks for testing!